modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -290,365 +345,498 @@ Removes a reference to an entity.
+* DYNAMICS:OPERATION_FAILED 
+* DYNAMICS:PARSE_ERROR 
+* DYNAMICS:UNAUTHORIZED 
+* DYNAMICS:UNKNOWN 
 
 
 [[retrieveMultipleByQuery]]
-== Retrieve Multiple By Query
+=== Retrieve Multiple By Query
 
 `<dynamics:retrieve-multiple-by-query>`
 
-
 Retrieve Multiple by DSQL Query:
 
-* Query Data using the Web API - https://msdn.microsoft.com/en-us/library/gg334767.aspx
+* Query Data using the Web API - `+https://msdn.microsoft.com/en-us/library/gg334767.aspx+`
 
 
+==== Parameters
 
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Datasense Query a| String |  The DSQL query that is going to be used for retrieve. The query will be transformed in a retrieve URL internally. |  | x
-| Page Size a| Number |  Number of items in a results page |  5000 |
+| Datasense Query a| String |  +++The DSQL query that is going to use for retrieve. The query will be transformed in a retrieve URL internally.+++ |  | x
+| Page Size a| Number |  +++Number of items in a results page+++ |  +++5000+++ | 
 | Streaming Strategy a| * <<repeatable-in-memory-iterable>>
 * <<repeatable-file-store-iterable>>
-* <<non-repeatable-iterable>> |  Configure if repeatable streams should be used and their behavior |  |
-| Target Variable a| String |  The name of a variable on which the operation's output will be placed |  |
-| Target Value a| String |  An expression that will be evaluated against the operation's output and the outcome of that expression will be stored in the target variable |  `#[payload]` |
+* non-repeatable-iterable |  +++Configure if repeatable streams should be used and their behavior+++ |  | 
+| Target Variable a| String |  +++The name of a variable to store the operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ |  `#[payload]` | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | 
 |===
 
-=== Output
+==== Output
 
-[cols=".^50%,.^50%"]
+[%autowidth.spread]
 |===
-| Type a| Array of Object
+|Type |Array of Object
 |===
 
 === For Configurations
 
-* <<dynamics-config>>
+* <<dynamics-config>> 
 
-=== Throws
+==== Throws
 
-* DYNAMICS:CONNECTIVITY
-* DYNAMICS:UNKNOWN
+* DYNAMICS:CONNECTIVITY 
+* DYNAMICS:INVALID_PAGE_SIZE 
+* DYNAMICS:NOT_FOUND 
+* DYNAMICS:OPERATION_FAILED 
+* DYNAMICS:PARSE_ERROR 
+* DYNAMICS:UNAUTHORIZED 
+* DYNAMICS:UNKNOWN 
 
 
 [[unauthorize]]
-== Unauthorize
+=== Unauthorize
 
 `<dynamics:unauthorize>`
 
-
++++
 Deletes all the access token information of a given resource owner ID so that it's impossible to execute any operation for that user without doing the authorization dance again.
```

Comment:
Hmm, this is an interesting sentence. 
What if it said something like:
Deletes all the access token information of a given resource owner ID so that it's impossible to execute any operation for the resource owner without performing the authorization exchange.